src/main.rs

```diff
@@ -260,7 +260,7 @@ fn create_app<'a, 'b>(
                 .long("seccomp")
                 .takes_value(true)
                 .possible_values(&["true", "false"])
-                .default_value("true"),
+                .default_value("false"),
```

Comment:
No you cannot do that, we chose to set the seccomp value to `true` on purpose.
You need to extend the seccomp filters if you want for your PR to work correctly.

Reply:
> No you cannot do that, we chose to set the seccomp value to `true` on purpose.
> You need to extend the seccomp filters if you want for your PR to work correctly.

Set seccomp to true change the default behaviour of the clh.  I think it should be set to false first.

Reply:
What do you mean it should be set to false first? We chose on purpose to set this option to true because this gives Cloud-Hypervisor users the maximum amount of security by default.
But because we also want to give developers an easy way to debug things they're working on, we added an option so that you disable it through the CLI.

Reply:
> What do you mean it should be set to false first? We chose on purpose to set this option to true because this gives Cloud-Hypervisor users the maximum amount of security by default.
> But because we also want to give developers an easy way to debug things they're working on, we added an option so that you disable it through the CLI.

I didn't see anyone need this maximum amount of security by default.
I just see it make clh cannot work in default.

Reply:
> I didn't see anyone need this maximum amount of security by default.
I just see it make clh cannot work in default.

Are you saying that it doesn't work with master (i.e. without any local changes?) otherwise this PR should be closed.